src/index.html

```diff
@@ -7,8 +7,60 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo"></div>
+      <nav class="nav">
+        <ul class="list">
+          <li class="li">
+            <a href="#" class="link is-active">
+              Apple
+            </a>
+          </li>
+          <li class="li">
+            <a href="#" class="link">
+              Samsung
+            </a>
+          </li>
+          <li class="li">
+            <a href="#"
+              class="link" data-qa="hover">
```

Comment:
fix indents in all places
```suggestion
            <a
              href="#"
              class="link"
              data-qa="hover"
            >
```